project.inlang/settings.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://inlang.com/schema/project-settings",
+    "sourceLanguageTag": "en",
+    "languageTags": [
+      "en", "es", "it", "ja", "pl", "pt-br", "ua", "zh-CN"
```

Comment:
@NilsJacobsen Hey, I think this is alright to merge as a useful tool, but could you rename `pt-br` to `pt-BR` in `languageTags`? I've just noticed in your editor that it doesn't pick up Portuguese translations, probably because the tags are case sensitive.